### PR TITLE
Bump rubygem-hammer_cli_katello to 0.16-master

### PR DESCRIPTION
**Pull request by akofink**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -94,6 +94,9 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/test
 
 %changelog
+* Wed Oct 24 2018 Andrew Kofink <akofink@redhat.com> 0.16.0-1.pre.master
```
[the remark this answers is not in the document]
Comment (akofink) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Wait, @ehelms what's going on? I thought it was X.Y.Z-rev

[the remark this answers is not in the document]
Comment (ekohl) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Well, the version here has no `.Z` so you have `X.Y-release`. In this case the release is `0.1`. By making all the prereleases `0.x` we ensure that a yum update will always prefer the final releases over prereleases.

**Comment by akofink** [comment]
Thanks! Updated

**Comment by ehelms** [comment]
This was handled as part of some updates to the prelease macro handling.

**Pull request closed without merging**